src/styles/variables/_variables.scss

```
@@ -0,0 +1,16 @@
+$fields-size: 100px;
+$font-family: arial, sans-serif;
+$font-size: 30px;
+$start-day:
+  ("mon": "0",
+  "tue": "101",
+  "wed": "202",
+  "thu": "303",
+  "fri": "404",
+  "sat": "505",
+  "sun": "606");
```

Hard-coded values again. Instead of this just use the index (order) of the days and then you will be able to calculate proper margins ;)
```suggestion
$start-day:
  ("mon": 0,
  "tue": 1,
  "wed": 2,
  "thu": 3,
// etc ...
```